apps-rendering/package.json

```diff
@@ -42,10 +42,10 @@
 		"@emotion/server": "^11.4.0",
 		"@guardian/apps-rendering-api-models": "6.2.0",
 		"@guardian/bridget": "^2.0.0",
-		"@guardian/cdk": "^50.10.10",
+		"@guardian/cdk": "50.10.10",
 		"@guardian/content-api-models": "17.8.0",
 		"@guardian/content-atom-model": "^3.4.0",
-		"@guardian/eslint-config-typescript": "^5.0.0",
+		"@guardian/eslint-config-typescript": "^7.0.0",
```

Comment:
Should we pin this?

Reply:
probably? AR deps probably all want pinning but it doesn't affect the yarn stuff, so i didn't want to start down that road in here. makes sense though

Reply:
Leave it with us 🙇